src/main.ts

```diff
@@ -78,7 +78,12 @@ export { type Base, type BaseProperties, BaseClient } from './components/base';
  * @group Raw Protobufs
  */
 export { default as boardApi } from './gen/component/board/v1/board_pb';
-export { type Board, BoardClient } from './components/board';
+export {
+  type Board,
+  BoardClient,
+  type DurationOptions,
```

Comment:
[minor] see comment related to naming

```suggestion
  type Duration,
```